src/PrestaShopBundle/Resources/config/services/adapter/product.yml

```
@@ -514,3 +514,29 @@ services:
       - '@prestashop.adapter.product.repository.product_multi_shop_repository'
       - '@prestashop.adapter.product.stock.repository.stock_available_multi_shop_repository'
       - '@prestashop.adapter.shop.repository.shop_repository'
+
+  prestashop.adapter.product.command_handler.update_product_handler:
+    class: PrestaShop\PrestaShop\Adapter\Product\CommandHandler\UpdateProductHandler
+    arguments:
+      - '@prestashop.adapter.product.update.product_updatable_property_filler'
+      - '@prestashop.adapter.product.repository.product_multi_shop_repository'
+      - '@prestashop.adapter.product.update.product_indexation_updater'
+    tags:
+      - name: tactician.handler
+        command: PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductCommand
+
+  prestashop.adapter.product.update.product_updatable_property_filler:
+    class: PrestaShop\PrestaShop\Adapter\Product\Update\ProductUpdatablePropertyFiller
+    arguments:
+      - !tagged core.product_updatable_property_filler
```

I we decide to simplify and rename the interface I guess the tag could also be shortened
```suggestion
      - !tagged core.product_filler
```

---

I made some other comments regarding the interface naming, this tag renaming would make even more sense if the interface is renamed